SonarQube.Client.Tests/SonarQube.Client.Tests.csproj

```diff
@@ -7,7 +7,6 @@
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <RootNamespace>SonarQube.Client.Tests</RootNamespace>
-    <LangVersion>6</LangVersion>
```

Comment:
I can't think of any reason for setting this to restrict language features to a lower level than that supported by the target framework so I've removed it. It was preventing me from using pattern matching in the changes of the `RequestFactory`.